### DISCUSSION TITLE
Can we create our own components and have it work with Landamic blocks?

### DISCUSSION THREAD

**Discussion by paulpreibisch** [lucky-media/landtamic#36] [General]
I am considering purchasing this. I am a Laravel Developer, so everything I assume is modifiable.  Can I for instance create my own Set, and then add it to the page using Landamics page builder?

**Comment by lokmanm** [comment]
> I am considering purchasing this. I am a Laravel Developer, so everything I assume is modifiable. Can I for instance create my own Set, and then add it to the page using Landamics page builder?

Hey @paulpreibisch,

Yes, that's where Statamic comes into play. You can easily create a new set in the `Builder` fieldset and then make the corresponding views and it should _work_ ™️. ( assuming you set it up correctly )

Let me know if you need any help!
